x-pack/plugins/aiops/common/index.ts

```diff
@@ -19,4 +19,4 @@ export const PLUGIN_NAME = 'AIOps';
  * This is an internal hard coded feature flag so we can easily turn on/off the
  * "Explain log rate spikes UI" during development until the first release.
  */
-export const AIOPS_ENABLED = false;
+export const AIOPS_ENABLED = true;
```

Comment:
Probably want to switch this back to false for now.

Reply:
Discussed with Pete and we're going to leave this enabled for now.